python/cugraph/cugraph/tests/dask/test_mg_renumber.py

```diff
@@ -176,3 +176,51 @@ def test_dask_pagerank(dask_client):
             err = err + 1
     print("Mismatches:", err)
     assert err == 0
+
+
+@pytest.mark.skipif(
+    is_single_gpu(), reason="skipping MG testing on Single GPU system"
+)
+@pytest.mark.parametrize("renumber", [False])
+def test_digraph_renumber_false(renumber, dask_client):
+    gc.collect()
+
+    input_data_path = (RAPIDS_DATASET_ROOT_DIR_PATH /
+                       "karate.csv").as_posix()
+    chunksize = dcg.get_chunksize(input_data_path)
+
+    ddf = dask_cudf.read_csv(
+        input_data_path,
+        chunksize=chunksize,
+        delimiter=" ",
+        names=["src", "dst", "value"],
+        dtype=["int32", "int32", "float32"],
+    )
+    dg = cugraph.DiGraph()
```

Comment:
Since `DiGraph` is deprecated, this should be:
```suggestion
    dg = cugraph.Graph(directed=True)
```